bin/stats.py

```diff
@@ -185,9 +185,9 @@ def value(x):
 
         verified = False
         comment = ""
-        if "verified" in problem.settings:
+        if problem.settings.verified is not None:
             verified = bool(problem.settings.verified)
-        if "comment" in problem.settings:
+        if problem.settings.comment is not None:
             comment = problem.settings.comment
```

Comment:
`None` coerces to `False`, and the default for `comment` is `""` which also coerces to `False`, so this can be shortened to:

```py
        verified = bool(problem.settings.verified)
        comment = problem.settings.comment or ""
```